plugins/notifiers/slack/client.go

```diff
@@ -136,24 +149,38 @@ func (n *notifier) sendRequest(req *http.Request) (*userResponse, error) {
 	return &result, nil
 }
 
+func getDefaultTemplate(messageType string) (string, error) {
+	content, err := os.ReadFile(fmt.Sprintf("plugins/notifiers/slack/templates/%s.json", messageType))
```

Comment:
should we [embed](https://pkg.go.dev/embed) the files or we add them when building the docker image?

```go
//go:embed plugins/notifiers/slack/templates/%s.json
var templateFiles embed.FS
```

Reply:
Done